public/images/icons/circle_check.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can encode and add these to icons.data.svg.css, to avoid separate requests for individual icons.